### PR TITLE
feat: JHA ContactInfo updates

### PR DESCRIPTION
**Pull request by Ian-c-cameron**
### Please check if the PR fulfills these requirements:

- [X] After these changes, the app was run and still works as expected
- [ ] Unit tests for these changes were added (if applicable)
- [X] All existing unit tests were run and still pass

### Please specify the type of change this PR introduces (Bug fix, feature addition, content update, chore, etc.):
Added tip box for phone number
Hide residential address when only FPC form is selected

### CODE REVIEW

File: jha/src/views/enrolment/ContactInfoPage.vue [hunk 1]
```diff
@@ -139,20 +139,34 @@
             </div>
           </div>
         </div>
-        <input type="checkbox" id="same-address-check" v-model="isMailSame">
-        <label for="same-address-check">This is my mailing address.</label>
+        <div v-if="!onlyFPC">
+          <input type="checkbox" id="same-address-check" v-model="isMailSame">
+          <label for="same-address-check">This is my mailing address.</label>
```
Comment (harrymaynard-maximus) [inline jha/src/views/enrolment/ContactInfoPage.vue:144]: Now that we have the "Checkbox" component in the common library, can you use that instead?

File: jha/src/views/enrolment/ContactInfoPage.vue [hunk 2]
```diff
@@ -268,31 +287,40 @@ export default {
...
-      resPostalCode: {
-        required,
-        bcPostalCodeValidator,
-      },
       phone: {
         phoneValidator,
       },
     };
+    if (!this.onlyFPC) {
+      let resValidations = {
```
Comment (harrymaynard-maximus) [inline jha/src/views/enrolment/ContactInfoPage.vue:302]: This should be a `const` since it's only being assigned an initial value.

**Review by harrymaynard-maximus — CHANGES_REQUESTED** [review]

File: jha/src/views/enrolment/ContactInfoPage.vue (hunk 2 above)
Comment (Ian-c-cameron) [reply jha/src/views/enrolment/ContactInfoPage.vue]: changed

File: jha/src/views/enrolment/ContactInfoPage.vue (hunk 1 above)
Comment (Ian-c-cameron) [reply jha/src/views/enrolment/ContactInfoPage.vue]: changed

**Review by harrymaynard-maximus — APPROVED** [review]
Looks good.

**Pull request merged**